### ISSUE TITLE
Player tenement: take or move particular cage multiplying it

### ISSUE THREAD

**Issue by Kastuk** [loiste-interactive/Obenseuer-Issues#592]
Renovation of makeshift heating in my appartment is completed, I did return into apartment, and then gonna to move away rat cages, which collided with the new barrel stove.
![2023-10-11 01_43_42-Greenshot](https://github.com/loiste-interactive/Obenseuer-Issues/assets/36314945/4b4d7f9d-7cee-4370-9659-240881a5672e)
Medium cage and small cage on top can be taken normal;
I tried to Take small cage on the floor (under second one);
Game did freeze for some time;
Multiplying of same cage in building inventory;
![2023-10-11 12_42_52-Greenshot](https://github.com/loiste-interactive/Obenseuer-Issues/assets/36314945/8d63f28b-e2c2-4d87-885b-bbd3736a64ca)

If I place it once, then second time, previously placed cage will disappear. Contain rats.
If I save and load game, multiplied cages become normal placeable. Contain none.

In the end I reload and tried to Move that cage after taking other ones and game freezed completelly.
![2023-10-11 02_44_26-Greenshot](https://github.com/loiste-interactive/Obenseuer-Issues/assets/36314945/a69faf75-4f0e-438d-a4e8-d955008ae51e)

[Before cage bug.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/12862930/Before.cage.bug.zip)
[Multiplied cage bug.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/12862931/Multiplied.cage.bug.zip)
[Stove fuel slot tests + multicage bug.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12862935/Stove.fuel.slot.tests.%2B.multicage.bug.log)

**Comment by Kastuk** [comment]
I got remember whats wrong with that cage: theres been a pair of glowing rats, and I did add one normal rat, which become glowing after some breeding. 
Then if I take this cage and place it again, theres will be only normal rats inside.

**Comment by Kastuk** [comment]
Aquarium with mixed breeding of normal and glowing fish eggs make same freeze at furniture Taking.

**Comment by Kastuk** [comment]
There's several tests with cage then aquarium.
[aquarium multibreed test.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/12869858/aquarium.multibreed.test.zip)

I did take mixed aquarium once and wait to end of freeze, than placed it back, all seems okay, no multiplying, fishes is same mixed breed. But console screen lost all text and not showing new lines, still can run commands.
Then I placed terrarium on top mismatched with aquarium and tried to take whole table with both glasscubes.
This freeze did overfill my poor memory and bring crash,
[Freezing cage and aquarium crash at the end.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12869851/Freezing.cage.and.aquarium.crash.at.the.end.log)
